### DISCUSSION TITLE
How to get pudb's version

### DISCUSSION THREAD

**Discussion by stdedos** [inducer/pudb#527] [Q&A] [answered]
Hello there! "For some reason", in order to have `pudb` always available (regardless of being in a `venv` or not), and to keep-up with the latest version, I am using this:

```bash
alias pudb='PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}'"$(realpath "${INSTALLS_DIR}/pudb")"'" python3 -m pudb.run '
```

_Being blinded by this_, and hitting a bug, I instinctively did:

```console
$ pudb --version
usage: run.py [options] [-m] SCRIPT-OR-MODULE-TO-RUN [SCRIPT_ARGS]
run.py: error: unrecognized arguments: --version
$ pudb -v
usage: run.py [options] [-m] SCRIPT-OR-MODULE-TO-RUN [SCRIPT_ARGS]
run.py: error: unrecognized arguments: -v
```

I was wondering - "what would be" the "correct" solution to my conundrum?
Should `run.py` just support a single `-v` argument, or a `--version` flag anywhere on the args as a "show version and quit" kind of thing?

**Comment by inducer** [comment, marked as answer]
I'd be open to adding a `--version` flag. That would only affect future versions, but that would probably be the cleanest. In the meantime, this might do the trick for you:
```shell
python -c 'import pudb; print(pudb.VERSION)' 
```

**Reply by stdedos** [reply]
I'd still like that `if sys.argv[1:] == ["-v"]` _(pseudo-code maybe)_ to mean `--version`, since typing `--version` it's _sometimes boring_ - but I totally get reserving `-v` for any future `--verbose` etc

**Reply by inducer** [reply]
Make a PR. I'll take a look.

**Reply by stdedos** [reply]
https://github.com/inducer/pudb/pull/528
